src/bokeh/server/views/autoload_js_handler.py

```diff
@@ -54,11 +54,14 @@ class AutoloadJsHandler(SessionHandler):
     '''
 
     def set_default_headers(self):
-        self.set_header("Access-Control-Allow-Headers", "*")
-        self.set_header('Access-Control-Allow-Methods', 'PUT, GET, OPTIONS')
         self.set_header("Access-Control-Allow-Origin", "*")
+        self.set_header("Access-Control-Allow-Headers", "*")
+        self.set_header("Access-Control-Allow-Credentials", "true")
 
     async def get(self, *args, **kwargs):
+        if self.request.cookies:
+            self.set_header("Access-Control-Allow-Origin", self.request.headers["Origin"])
```

Comment:
If credentials, i.e. cookies, are sent with the request, we cannot leave the allowed origin as wildcard `*`, but have to make it explicit.

Reply:
@philippjfr When this PR is merged, I'll send a PR to `panel` as well, since we override [`AutoloadJsHandler.get` there](https://github.com/holoviz/panel/blob/510d212840e1be51f37918bfbdbbd628fe88d16f/panel/io/server.py#L566-L572).